site/content/docs/5.0/layout/css-grid.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: docs
+title: CSS Grid
+description: Learn how to enable, use, and customize our alternate layout system built on CSS Grid with examples and code snippets.
+status: experimental
+group: layout
+toc: true
+---
+
+{{< callout info >}}
+- How to generate or create dynamic col classes above default of 12? Sass only?
+- Subgrid?
+- Shorthand for 2x2, 3x3, etc grids? .grid-2x12 (2 rows 12 cols)?
+- Wish we could do `cols="6"` and `[cols] { --columns: attr(cols number); }` but it's not supported.
```

Comment:
Note sure it has any sense for us but I solved in chaarts (not about grid but, well, charts) by using `style="--columns: 6;"` for example, which may sounds funny but is really great to work with (and exactly what the `style` attribute is useful for, IMHO).